$NetBSD: patch-ac,v 1.6 2008/10/17 11:03:43 tron Exp $

Fix build with GTK+ 2.13.* and newer. Patch taken from this bug report:

http://sourceforge.net/tracker/index.php?func=detail&aid=2070689&group_id=239&atid=100239

--- src/fe-gtk/xtext.h.orig	2008-02-24 04:48:02.000000000 +0000
+++ src/fe-gtk/xtext.h	2008-10-17 11:54:59.000000000 +0100
@@ -270,6 +270,6 @@
 xtext_buffer *gtk_xtext_buffer_new (GtkXText *xtext);
 void gtk_xtext_buffer_free (xtext_buffer *buf);
 void gtk_xtext_buffer_show (GtkXText *xtext, xtext_buffer *buf, int render);
-GtkType gtk_xtext_get_type (void);
+GType gtk_xtext_get_type (void);
 
 #endif
